### PR TITLE
Documentation(EJ2-887875): Getting started correction

### PR DESCRIPTION
**Pull request by NithyaSivaprakasam**

### CODE REVIEW

File: ej2-javascript/grid/js/getting-started.md [hunk 1]
```diff
@@ -69,9 +69,9 @@ Refer the below code to find location grid's script and style file.
 
 **Example:**
 
-> Script: `C:/Program Files (x86)/Syncfusion/Essential Studio/15.4.30/Essential JS 2/ej2-grids/dist/global/ej2-grids.min.js`
+> Script: `C:/Program Files (x86)/Syncfusion/Essential Studio/25.1.35/Essential JS 2/ej2-grids/dist/global/ej2-grids.min.js`
 >
-> Styles: `C:/Program Files (x86)/Syncfusion/Essential Studio/15.4.30/Essential JS 2/ej2-grids/styles/material.css`
+> Styles: `C:/Program Files (x86)/Syncfusion/Essential Studio/25.1.35/Essential JS 2/ej2-grids/styles/material.css`
 
 After copying the files, then you can refer the grid's scripts and styles into the `index.html` file.
```
Comment (MaithiliyK) [inline ej2-javascript/grid/js/getting-started.md:76]: is this sentence correct? Now we are adding the full dependency of grid

File: ej2-javascript/grid/js/getting-started.md [hunk 2]
```diff
@@ -81,19 +81,29 @@ The below html code example shows the minimal dependency of grid.
...
+          <link href="resources/notifications/styles/material.css" rel="stylesheet" type="text/css"/>
+          <link href="resources/splitbuttons/styles/material.css" rel="stylesheet" type="text/css"/>
+
+          <!-- Essential JS 2 Grid's dependent scripts -->
+          <script src="resources/scripts/ej2-base.min.js" type="text/javascript"></script>
+          <script src="resources/scripts/ej2-data.min.js" type="text/javascript"></script>
+          <script src="resources/scripts/ej2-popups.min.js" type="text/javascript"></script>
+
+          <!-- Essential JS 2 Grid's global script -->
+          <script src="resources/scripts/ej2-grids.min.js" type="text/javascript"></script>
```
Comment (MaithiliyK) [inline ej2-javascript/grid/js/getting-started.md:106]: initially we are using ej2-grids.min.js file but later we are using ej2-min.js file which is correct

**Review by MaithiliyK — COMMENTED** [review]
address my review comments

File: ej2-javascript/grid/js/getting-started.md (hunk 2 above)
Comment (NithyaSivaprakasam) [reply ej2-javascript/grid/js/getting-started.md:108]: Both are correct mam. When we use ej2-grids.min.js, it includes only the dependencies required for the grid. However, if we opt for ej2-min.js, it includes all dependencies. This helps us avoid dependency errors like the reported paging issue
image
already added this point mam

File: ej2-javascript/grid/js/getting-started.md (hunk 1 above)
Comment (NithyaSivaprakasam) [reply ej2-javascript/grid/js/getting-started.md]: Modified the sentance mam

File: ej2-javascript/grid/js/getting-started.md [hunk 3]
```diff
@@ -119,58 +197,145 @@ Refer the grid's CDN links as below
...
+</html>
+```
+
+Based on your requirement, additional dependencies can be included as needed. In this documentation, the common **ej2.min.js** and **material.css** are used, which include all the Essential JS 2 components and their dependent scripts and theme files.
+
 ## Adding Grid control
 
 Now, you can start adding Grid control in the application. For getting started, add a **div** element for Grid control in **index.html**. Then refer the **index.js** file into the **index.html** file.
 
-In this document context we are going to use **ej2.min.js** which includes all the Essential JS 2 components and its dependent scripts.
```
Comment (MaithiliyK) [inline ej2-javascript/grid/js/getting-started.md:152]: why this line is removed

Comment (NithyaSivaprakasam) [reply ej2-javascript/grid/js/getting-started.md:152]: Added this point before Adding grid control topic
image

**Review by MaithiliyK — APPROVED** [review]
changes are fine

**Pull request merged**